src/storybook/stories/components/button.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { boolean } from "@storybook/addon-knobs"
+import ButtonCo from "../../../components/buttonCo"
+
+const cases = ["AlertCo", "ButtonCo"]
+const stories = storiesOf("ButtonCo", module)
+cases.forEach((label) =>
```

Comment:
Do you think that be good add braces for more readability? 

```suggestion
cases.forEach((label) => {
```